Wrappers/Python/test/test_quality_measures.py

```diff
@@ -74,12 +88,34 @@ def test_psnr1(self):
         res2 = peak_signal_noise_ratio(self.id_coins.as_array(), self.id_coins_noisy.as_array())
         np.testing.assert_almost_equal(res1, res2, decimal=3)
         
+    @unittest.skipIf((not has_skimage) or version.parse(np.version.version) < version.parse("1.13"), "Skip test with numpy {} < 1.13 or has_skimage {}".format(np.version.version, has_skimage))
```

Comment:
what's the test on numpy version for?

Reply:
I have removed it - don't know why it was there